plugins/rikki/heroeslounge/models/Match.php

```diff
@@ -162,6 +166,14 @@ public function getAcceptedCasters()
         });
     }
 
+    public function getAcceptedOrPendingCasters()
+    {
+        return $this->casters
+        ->filter(function ($item) {
+            return $item->pivot->approved == 0 || $item->pivot->approved == 1;
+        });
+    }
+
```

Comment:
You can split this into two functions (one of which exists already), or just do this in the function itself (although the first approach would be better)